class-00/lab-c/lab/starter-code/styles/modules.css

```diff
@@ -0,0 +1,153 @@
+@import url('https://fonts.googleapis.com/css?family=Abel|Oswald:700|Yellowtail|Patrick+Hand');
+
```

Comment:
I would argue that some of these things would go in your layout file. For example, when you are floating or defining height/width that would be considered layout styling. You will find that a lot of this is up to your discretion, so as long as you are consistent and can defend your choices, then you can justify it.